packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -49,11 +49,19 @@ export default class BookReaderWrapper extends Component {
       searchInsideUrl: '/fulltext/inside.php',
       initialSearchTerm: null,
       imagesBaseURL: '/bookreader/BookReader/images/',
-      //TODO-ISA what is going on here, a comment would be useful ? (same on bookreader-wrapper-main and bookreader-wrapper-jsia)
+      useSrcSet: false,
+      /**
+       * Needed bypass to generate Image URL with scale factor.
+       * We must eliminate sooner than later to allow BookReader full image fetching control
+       * @param {Number} index - page index
+       * @param {Number} reduce - image size scale factor
+       * @param {Number} rotate - degrees of rotation
+       */
       getPageURI: (index, reduce = 1, rotate = 0) => {
-        let uri = originalGetPageURI.call(br, index, reduce, rotate);
+        const brReduce = this.bookreader.reduce || reduce;
```

Comment:
QUESTION: So just checking: `this.bookreader.reduce` will now take precedence over any passed-in `reduce`, right?

Reply:
yes.  this URL string method bypass is NOT the most optimum. in fact we should not even have it at all - but at the moment, we must hold.

We never want the URL to be requested without its image scale, unless you are extremely zoomed, in - that case we handle well enough.

BR 1up mode - sets reducer & reduction factors that allow zooming - and we always want to prefer the calculated image scale at this level. like we did for 2up mode.